packages/flutter_tools/templates/module/ios/host_app_ephemeral_cocoapods/Podfile.copy.tmpl

```diff
@@ -0,0 +1,6 @@
+platform :ios, '8.0'
+
+target 'Runner' do
+  flutter_application_path = '../'
+  eval(File.read(File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')))
+end
```

Comment:
Do we still need to disable bitcode for included third-party pods?

Reply:
It seems not.